docs/topics_5/rn-new-features.adoc

```diff
@@ -3,15 +3,68 @@
 [id='rn-new-features_{context}']
 = New features
 
-This section describes the new features of the {ProductName} {DocInfoProductNumber}.
+This section describes the new features of the {ProductName} ({ProductShortName}) {DocInfoProductNumber}.
 
-Updated {WebName} interface::
-The {WebName} has been redesigned with PatternFly 4. In addition to its general improved appearance, the {WebName} adds elements for project creation and management and updated transformation target icons.
+[id='updated-web-console-interface_{context}']
+== Updated {WebName} interface
+The {WebName} has been redesigned with PatternFly 4. The updated {WebName} has improved project management features and additional transformation targets.
 
-{ProductName} Operator for OpenShift Container Platform::
-The current release includes the {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.5 and later.
+[id='mta-operator-for-ocp_{context}']
+== {ProductName} Operator for OpenShift Container Platform
+The release includes the {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.6 and later.
 
-Greater support for migrating from Spring Boot to Quarkus::
-The current release provides greater support for migrating from Spring Boot to Quarkus with the addition of 22 rules.
-+
-The new rules focus on the Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+[id='mta-installation-in-multiple-namespaces_{context}']
+== Installation in multiple namespaces
+You can now install the {ProductShortName} in multiple namespaces.
+
+[id='greater-support-for-spring-boot-to-quarkus_{context}']
+== Greater support for migrating from Spring Boot to Quarkus
+The release includes more rules for migrating from Spring Boot to Quarkus.
+
+The new rules identify Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+
+The rules identify the following technologies, which have equivalent Quarkus extensions:
```

Comment:
You can remove `, which have equivalent Quarkus extensions` because you already mentioned the equivalent extensions in the previous paragraph.